bindings/python/__init__.py

```diff
@@ -0,0 +1,23 @@
+import typing
+
+try:
```

Comment:
Why do we have a try catch blog at the init?

Reply:
Following https://stackoverflow.com/a/47645630/4565794, but keeping minimal effort. A pure python fallback is too much work.

Reply:
If you're just re-raising the import error, why the try/except block? If you didn't catch it, it would look exactly the same to the user. But with this try/except block any other type of error raised anywhere in this code is just silently ignored even though initialisation probably failed due to it.

Reply:
I removed the `try...catch`. 

(I had ideas of working around typing with that and dummy objects, which turned out to be too cumbersome enough to give up quickly.)